bucket/robo3t.json

```diff
@@ -16,11 +16,15 @@
             "Robo 3T"
         ]
     ],
+    "checkver": {
+        "github": "https://github.com/Studio3T/robomongo",
+        "regex": "robo3t-(?<version>[\\d.]+)-windows-x86_64-(?<hash>[0-9a-f]+)"
```

Comment:
It's not a hash though

Commit would maybe be more appropriate?

https://github.com/Studio3T/robomongo/commit/e6ac9ec51319253850bac47b1f5795f1cb6622f7

Reply:
Otherwise LGTM and its ready to merge!

Reply:
Ok, if you want to be technical, it's the `Short SHA-1` with the H in that standing for `hash`. Ref: https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection

If you really think it's required to change it, I'll do so, but I think that's a bit pedantic.

Reply:
IMHO it becomes a bit more clearer

Maybe commitHash if you would prefer that?

Reply:
Because TBH when we read hash, we usually instantly think the hash of the file rather than the hash of the commit in question

Reply:
Done.